frontend/openchat-client/src/openchat.ts

```diff
@@ -4172,15 +4188,15 @@ export class OpenChat extends OpenChatAgentWorker {
         localMessageUpdates.markThreadSummaryUpdated(threadRootMessageId, summary);
     }
 
-    broadcastMessageRead(chat: ChatSummary, messageId: bigint): void {
+    private broadcastMessageRead(chat: ChatSummary, messageId: bigint): void {
```

Comment:
broadcast seems like the wrong name for this function - it only sends to one user